Assets/Scripts/LevelData.cs

```diff
@@ -6,11 +6,18 @@
 /// </summary>
 public class LevelData : MonoBehaviour
```

Comment:
Should try and give it a namespace and put it in a sub-folder for scripts.

Reply:
ai